packages/cli/package.json

```diff
@@ -44,7 +44,8 @@
         "picocolors": "^1.0.0",
         "prompts": "^2.4.2",
         "vite": "^4.5.0",
-        "vite-plugin-externals": "^0.6.2"
+        "vite-plugin-externals": "^0.6.2",
+        "yup": "^1.3.2"
```

Comment:
Please use `zod`, I will remove `yup` from `web-app` soon ;)

Reply:
ah alright, i reckoned it would be yup as it was in the web-app. Ill get to it